cmip6_downscaling/methods/bcsd/flow.py

```diff
@@ -19,6 +19,7 @@
     rechunk,
     run_analyses,
 )
+from cmip6_downscaling.workflows.utils import rechunk_zarr_array_with_caching
```

Comment:
please move this task from workflow/utils to methods/common/ and rename the task to `rechunk`